docs/rst/tutorials/tools/monitor_backend/monitor_backend.rst

```diff
@@ -0,0 +1,265 @@
+This class overrides the default PhysicalListener callbacks, which allows the execution of routines in case of an event.
+
+.. literalinclude:: /../code/monitor/src/monitor.cpp
+    :language: C++
+    :lines: 214-224
+
+Within the PhysicalListener class, we can override several callback to adapt how the monitor application reacts to some events.
+These overridden callbacks are:
+
+*   ``on_host_discovery()`` allows the definition of a series of actions when a new host is detected.
+
+    .. literalinclude:: /../code/monitor/src/monitor.cpp
+        :language: C++
+        :lines: 226-240
+
+*   ``on_user_discovery()`` detects when a new user is discovered.
+
+    .. literalinclude:: /../code/monitor/src/monitor.cpp
+        :language: C++
+        :lines: 242-256
+
+*   ``on_process_discovery()`` involves when a new process is discovered.
+
+    .. literalinclude:: /../code/monitor/src/monitor.cpp
+        :language: C++
+        :lines: 258-272
+
+*   ``on_topic_discovery()`` is called when a new Topic is discovered.
+
+    .. literalinclude:: /../code/monitor/src/monitor.cpp
+        :language: C++
+        :lines: 274-294
+
+*   ``on_participant_discovery()`` is called when a new participant is discovered.
+
+    .. literalinclude:: /../code/monitor/src/monitor.cpp
+        :language: C++
+        :lines: 296-312
+
+*   ``on_datareader_discovery()`` and ``on_datawriter_discovery()`` involves when a new DataReader or DataWriter respectively are discovered.
+
+    .. literalinclude:: /../code/monitor/src/monitor.cpp
+        :language: C++
+        :lines: 314-348
+
+Finally, the monitor application is initialized and run in ``main`` function.
+
+.. literalinclude:: /../code/monitor/src/monitor.cpp
+    :language: C++
+    :lines: 360-374
+
+CMakeLists.txt
+^^^^^^^^^^^^^^
+
+Include at the end of the `CMakeList.txt` file you created earlier the following code snippet.
+This adds all the source files needed to build the executable, and links the executable and the library together.
+
+.. literalinclude:: /../code/monitor/CMakeLists.txt
+    :language: cmake
+    :lines: 40-55
+
+Running the application
+-----------------------
+
+At this point the project is ready for building, compiling and running the application.
+From the base workspace directory (``ros_ws``), run the following commands.
+
+.. code-block:: bash
+
+    colcon build
+    source install/setup.bash
+    ros2 run monitor monitor
+
+Then open two more terminals and load the Vulcanexus environment.
+Then, in one of them run a ``talker`` and in the other one a ``listener`` of the ``demo_nodes_cpp`` ROS 2 package, available in the Vulcanexus Desktop distribution.
+
+*   Terminal 1:
+
+    .. code-block:: bash
+
+        source /opt/vulcanexus/humble/setup.bash
+        <enable_statistics>
```

Comment:
I would add this the explicit command required
`export FASTDDS_STATISTICS="HISTORY_LATENCY_TOPIC;PUBLICATION_THROUGHPUT_TOPIC;PHYSICAL_DATA_TOPIC"`